.eslintrc.json

```diff
@@ -13,6 +14,7 @@
     "linebreak-style": ["error", "unix"],
     "no-tabs": "error",
     "no-trailing-spaces": "error",
+    "no-var": "error",
```

Comment:
I think you need to add more ES6 specific rules here.

Reply:
yep but I don't know all the rules 😟

Reply:
Just look into what we use in Bootstrap, search for `prefer` in our config file :)